Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
+import './reset.css';
 import './App.css';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import CardContainer from './components/CardComponents/CardContainer';
+import Footer from './components/FooterComponents/Footer';
+
+// <h3>Welcome to React Social Card!</h3>
+// <p>
+//   Begin by exploring the `components` directory. You'll notice we have a
+//   few files that we've already included in there to get you started right
+//   away building components. You'll need to make sure you include your
+//   components that you build in this file to watch your app come to life
+// </p>
+// <p>
+//   <strong>
+//     Don't forget to `default export` your components and import them here
+//     inside of this file in order to make them work.
+//   </strong>
+// </p>
 
 const App = () => {
   return (
     <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+
```

Comment:
good job. Glad to see the concept of importing components and calling them in the return here was easy for you, but what I really appreciate is that you are still keeping the lessons from semantic HTML in mind. Instead of simple div like most people you thought to use an article. Very well done